lrpar/src/lib/ctbuilder.rs

```diff
@@ -1547,6 +1528,10 @@ where
     }
 }
 
+fn indent(s: &str, indent: &str) -> String {
```

Comment:
Let's document this one. I suspect we should also reorder the parameters (I misread the calls above!).

Reply:
Is it okay if I reorder the parameters in a separate PR?

Since this function is copy/paste from nimbleparse I'd want to update that as well, and I fear updating all the callers over seems like it might make these patches a bit more messy.